lib/target_mvd/targets.ex

```diff
@@ -47,9 +47,16 @@ defmodule TargetMvd.Targets do
   end
 
   def create_target(attrs \\ %{}) do
-    %Target{}
-    |> Target.changeset(attrs)
-    |> Repo.insert()
+    user = User |> Repo.get!(attrs["user_id"]) |> Repo.preload(:targets)
+    targets = Enum.count(user.targets)
+
+    if targets >= 10 do
+      {:error, :maximum_reached}
```

Comment:
This could return an error message as well, so it's easier to render the error view afterwards. 
`{:error, :maximum_reached, "Target maximum limit reached"}` what do you guys think?

Reply:
Definitely yes! Prefer to have the message in the context for now rather than in the controller

Reply:
Agreed, remember to use `gettext` to translate it